core/whistle_media-1.0.0/src/wh_media_recording.erl

```diff
@@ -355,7 +366,7 @@ handle_cast({'gen_listener',{'is_consuming', 'true'}}, #state{record_on_answer='
                                                              }=State) ->
     start_recording(Call, MediaName, TimeLimit, <<"wh_media_recording">>, SampleRate, RecordMinSec),
     lager:debug("started the recording"),
-    {'noreply', State};
+    {'noreply', State#state{is_recording='true'}};
```

Comment:
@jamesaimonetti 
My settings are:
- "record_on_answer": false;
-  "store_recordings": true,
-  "third_party_bigcouch_host": "127.0.0.1",
-  "third_party_bigcouch_port": 5984

Without changing "is_recording" to 'true' there was even no attempts to save the call.
In the other words the call was recorderd but left in /tmp untouched.
I am talking about all calls, not only forwarded/transferred.
So if it is way too early to set "is_recording" to 'true' I need to try to find out why we do not see RECORD_START later. 
Will do..